netlistsvg/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = '%s_%04i_%s'|format(GIT_DESCRIBE_TAG|replace('v','') or '0.X', GIT_DESCRIBE_NUMBER|int, GIT_DESCRIBE_HASH or 'gUNKNOWN') %}
+
+package:
+  name: netlistsvg
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/nturley/netlistsvg.git
+  git_rev: master
+
+build:
+  # number: 201803050325
+  number: {{ environ.get('DATE_NUM') }}
+  # string: 20180305_0325
+  string: {{ environ.get('DATE_STR') }}
+  script_env:
+    - CI
+    - TRAVIS
+
+requirements:
+  build:
+    - nodejs
+  host:
+    - nodejs
```

Comment:
One of `build: nodejs` and `host: nodejs` should be enough. From the `build.sh` script that uses `$PREFIX` instead of `$BUILD_PREFIX` I think the `build: nodejs` is the one that isn't really used.

Reply:
`build: nodejs` removed. It truly isn't used.